### ISSUE TITLE
Spyder problem when parsing an xml file

### ISSUE THREAD

**Issue by GaoYang-Thu** [spyder-ide/spyder#7191]
## Description

**What steps will reproduce the problem?**

1. try to parse an XML file using xml.etree.Element
2. 
3. 

**What is the expected output? What do you see instead?**
I expected to see the elements of the xml file.
However, I saw "there is no element" instead and Spyder dialogue window appeared, saying "there is a problem.."

**Please provide any additional information below**

  File "C:\Users\YangGao\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 1516, in go_to_error
    self.edit_goto.emit(osp.abspath(fname), int(lnb), '')
ValueError: invalid literal for int() with base 10: ''

## Version and main components

* Spyder Version: 3.2.3 
* Python Version: 3.6.4
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.22.0 (OK)
numpy >=1.7      :  1.14.2 (OK)
sphinx >=0.6.6   :  1.6.6 (OK)
rope >=0.9.4     :  0.10.7 (OK)
jedi >=0.9.0     :  0.11.1 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.27.3 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.2.1 (OK)
pylint >=0.25    :  1.8.2 (OK)

```

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.2.8). Please update by opening the Anaconda Prompt and running there

    conda update spyder

**Comment by CAM-Gerlach** [comment]
Duplicate of #5402 , fixed in Spyder 3.2.5—your version is well over half a year out of date, FYI.
